src/Enqueue/Theme/EnqueueTheme.php

```diff
@@ -77,6 +79,36 @@ public function enqueueStylesLocal()
 		$this->enqueueStyles();
 	}
 
+	/**
+	 * Method that returns frontend script for captcha if settings are correct.
+	 *
+	 * @return mixed
+	 */
+	public function enqueueScriptsCaptcha()
+	{
+		// Check if Captcha data is set and valid.
+		$isSettingsGlobalValid = \apply_filters(SettingsCaptcha::FILTER_SETTINGS_GLOBAL_IS_VALID_NAME, false);
+
+		// Bailout if settings are not ok.
+		if (!$isSettingsGlobalValid) {
+			return null;
```

Comment:
`wp_enqueue_scripts` is an action, and doesn't return anything, so you can just do a `return;` (void).